test/regressionmodel.jl

```diff
@@ -15,4 +15,4 @@ StatsAPI.modelmatrix(::MyRegressionModel) = [1 2; 3 4]
     @test crossmodelmatrix(m) isa Symmetric
 end
 
-end # module TestRegressionModel
\ No newline at end of file
+end # module TestRegressionModel
```

Comment:
You should also be able to revert changes to this file and the next one.

Reply:
the missing terminal newlines? I think those should be required FWIW

Reply:
done